tests/integration/test_trustedactivites.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from tests.integration.conftest import assert_successful_response
+
+
+@pytest.mark.integration
+class TestTrustedActivities:
+    @pytest.fixture(scope="module")
+    def trusted_activity(self, connection):
+        return connection.trustedactivities.create("DOMAIN", "test.com")
+
+    def test_get_all_trusted_activities(
```

Comment:
does the mock server complain about unrecognized query params (i.e. us sending pgNum and pgSize even though they aren't defined yet?) The _real_ server won't of course, but I wasn't sure about the mock.

Reply:
I haven't added the pgSize param to the mock-servers yet but it doesn't seem to complain about it as an unrecognized query param when sent in an integration test.